src/options.cpp

```diff
@@ -1703,7 +1703,12 @@ void options_manager::add_options_world_default()
        );
 
     add( "SPAWN_DENSITY", "world_default", translate_marker( "Spawn rate scaling factor" ),
-         translate_marker( "A scaling factor that determines density of monster spawns." ),
+         translate_marker( "A scaling factor that determines density of monster spawns (except animals)." ),
+         0.0, 50.0, 1.0, 0.1
+       );
+
+    add( "SPAWN_ANIMAL_DENSITY", "world_default", translate_marker( "Animal spawn rate scaling factor" ),
+         translate_marker( "A scaling factor that determines density of wild, formerly domesticated and mutated animal spawns." ),
```

Comment:
```suggestion
         translate_marker( "A scaling factor that determines density of wild, formerly domesticated and mutated animals spawns." ),
```

Reply:
That one's actually correct as a singular

Reply:
Enumeration of 1) wild, 2) formerly domesticated, and 3) mutated should be "animal"?

Reply:
"animal spawn" is kinda treated as one thing in this case so the plural would be "animal spawns."  Also it just sounds right to me. 

Either that or add an apostrophe: "animal's spawns" but that's a little awkward.